### PR TITLE
Add route to get files in base64

### PR DESCRIPTION
**Pull request by nh758**
see https://github.com/digi-serve/ab_service_file_processor/pull/34

### CODE REVIEW

File: api/controllers/file_processor/file-base64.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * @apiParam {string} ID file uuid
+ * @apiQuery {boolean} mobile whether to reutrn downscaled images for mobile
+ * @apiUse successRes
+ * @apiSuccess (200) {Object} data
+ * @apiSuccess (200) {string} data.image base64
+ */
+
+var inputParams = {
+   ID: { string: { uuid: true }, required: true },
+   mobile: { boolean: true },
```
Comment (Hiro-Nakamura) [inline api/controllers/file_processor/file-base64.js:17]: we should probably either specify `required` or `optional` here.

File: README.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ ___
   - [User Applications](#User-Applications)
 - [File](#File)
   - [Get a File](#Get-a-File)
+  - [Get a File as base64](#Get-a-File-as-base64)
```
Comment (Hiro-Nakamura) [inline README.md:42]: wow, look at all this documentation!  How long has this been here?

**Review by Hiro-Nakamura — APPROVED** [review]
This looks good.

Didn't know our `README.md` had so much documentation.

File: api/controllers/file_processor/file-base64.js (hunk 1 above)
Comment (nh758) [reply api/controllers/file_processor/file-base64.js:17]: good catch, yes meant it to be optional

File: README.md (hunk 2 above)
Comment (nh758) [reply README.md:42]: I added it earlier this year. Uses [apidoc](https://www.npmjs.com/package/apidoc) to generate, based on comments in each controller.

**Review by wongpratan — APPROVED** [review]

**Pull request merged**